tests/MRA.Jobs.Api/Application.IntegrationTests/Jobs/Command/CreateJobVacancyCommandTest.cs

```diff
@@ -61,6 +64,49 @@ public async Task CreateJobVacancyCommand_CreatingJobVacancyWithTask_Success()
 
         response.Should().NotBeNull();
     }
+    [Test]
+    public async Task CreateJobVacancyCommand_CreatingJobVacancyWithTask_ConflictException()
+    {
+        RunAsReviewerAsync();
+        var jobVacancy = new CreateJobVacancyCommand
```

Comment:
Just create vacancy from EF, do not use Questions and Tasks, you don;t need them to test slug